diff --git a/src/documents/consumer.py b/src/documents/consumer.py
index 9329347f1..9c2f99052 100644
--- a/src/documents/consumer.py
+++ b/src/documents/consumer.py
@@ -857,7 +857,8 @@ class ConsumerPlugin(
             self.log.debug(f"Creation date from parse_date: {create_date}")
         else:
             stats = Path(self.input_doc.original_file).stat()
-            create_date = timezone.make_aware(
-                datetime.datetime.fromtimestamp(stats.st_mtime),
+            create_date = datetime.datetime.fromtimestamp(
+                stats.st_mtime,
+                tz=timezone.get_current_timezone(),
             )
             self.log.debug(f"Creation date from st_mtime: {create_date}")
